core/spec/services/spree/cart/create_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+module Spree
+  describe Cart::Create do
+    subject { described_class }
+
+    let(:user) { create :user }
+    let(:store) { create :store }
+    let(:currency) { 'USD' }
+
+    it 'create an order' do
```

Comment:
@gingray let's test
* how it behaves when `order_params` are passed vs when those aren't passed
* when no store and currency aren't passed as params
* does it generate an order number properly